OpenRA.Game/Player.cs

```diff
@@ -130,20 +130,27 @@ public Player(World world, Session.Client client, PlayerReference pr)
 			fogVisibilities = PlayerActor.TraitsImplementing<IFogVisibilityModifier>().ToArray();
 
 			// Enable the bot logic on the host
-			IsBot = botType != null;
-			if (IsBot && Game.IsHost)
+			ActivateBotOnHost(botType);
+
+			stanceColors.Self = ChromeMetrics.Get<Color>("PlayerStanceColorSelf");
+			stanceColors.Allies = ChromeMetrics.Get<Color>("PlayerStanceColorAllies");
+			stanceColors.Enemies = ChromeMetrics.Get<Color>("PlayerStanceColorEnemies");
+			stanceColors.Neutrals = ChromeMetrics.Get<Color>("PlayerStanceColorNeutrals");
+		}
+
+		public void ActivateBotOnHost(string botType)
```

Comment:
The `OnHost` in the name here is a bit confusing – based on the name I would expect you to be passing the host as a parameter.  `ActivateBot` is fine.